webui/index.js

```diff
@@ -13328,7 +13328,7 @@ function getIbanFromPayto(url) {
 }
 var maybeRootPath = "https://bank.taler.example.org/demobanks/default/";
 function getBankBackendBaseUrl() {
-  const overrideUrl = localStorage.getItem("bank-base-url");
+  const overrideUrl = bankUiSettings.bankBaseUrl;
```

Comment:
This lead to an error in the webui as long as `maybeRootPath` was not set correctly.